src/lib/render.py

```diff
@@ -95,9 +119,8 @@ def draw_catalogue(
 def reset_catalogue(catalogue_set):
     if catalogue_set is not None:
         catalogue_set.remove()
-        catalogue_set = None
```

Comment:
was it not important to have the catalogue_set be None here?

Reply:
I changed to just returning None as its no different to setting catalogue_set = None and returning that

Reply:
But this means catalogue_set will have a dangling reference to the catalogue set still. I think if we don't set it to None here the garbage collector won't remove it